code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -422,9 +422,11 @@
 	process_flags = ORGANIC | SYNTHETIC
 
 /datum/reagent/stableslimetoxin/on_mob_life(mob/living/carbon/human/H)
-	..()
 	if(!istype(H))
 		return
+	if(volume < 1)
```

Comment:
`on_mob_life()` handles removing the reagent and a bunch of fancy stuff up the chain. You'll want to still run `..()` if you're returning early.

Reply:
foxed